f2/src/forms/EvidenceInfoForm.js

```diff
@@ -68,10 +68,29 @@ export const EvidenceInfoForm = (props) => {
       e.target.value = '' // clear the file input target, to allow the file to be chosen again
       return
     }
-    setStatus('fileUpload.added')
-    setFiles(files.concat(e.target.files[0]))
-    setFileDescriptions(fileDescriptions.concat(''))
-    e.target.value = '' // clear the file input target, to allow the file to be removed then added again
+    if(file.type.indexOf("image") !== -1) {
+      let img = new Image()
+      img.src = window.URL.createObjectURL(file)
+      img.onload = () => {
+        if (img.width < 128 || img.height < 128){
```

Comment:
Now that the refactored form has been merged we should use the modal to display error messages here instead of alert.